### PR TITLE
RDoc-543 conflicts view Resolve all conflicts

### PR DESCRIPTION
**Pull request by IdanHaim**

### CODE REVIEW

**Comment by arekpalinski** [comment]
Can you put a smaller image `conflicts_view_5.png` ? I mean to change the browser size when you do a screenshot - most of the image content is filled by empty space while text is hardly visible

File: Documentation/3.5/Raven.Documentation.Pages/studio/overview/documents/conflicts-view.markdown [hunk 1]
```diff
@@ -24,4 +24,21 @@ You can resolve the conflict in the studio by editing the conflicted parts and s
 
 Then the conflict will disappear:
 
-![Figure 5. Studio. No conflicts.](images/conflicts_view_5.png)  
+![Figure 5. Studio. No conflicts.](images/conflicts_view_5.png)
+
+Another way to solve all the issues in this page is to press the Reolve all conflicts button and chose how to resolve them:
```
Comment (arekpalinski) [inline Documentation/3.5/Raven.Documentation.Pages/studio/overview/documents/conflicts-view.markdown]: typo: Reolve

File: Documentation/3.5/Raven.Documentation.Pages/studio/overview/documents/conflicts-view.markdown [hunk 2]
```diff
@@ -24,4 +24,21 @@ You can resolve the conflict in the studio by editing the conflicted parts and s
...
+<span class="float-left">![Figure 7. Studio. resolve_conflicts_options.](images/resolve_conflicts_options.png)
+
+* `Resolve with Local` if you want to resolve the conflicts in favor of local version,
+* `Resolve to newest remote` if you want to resolve the conflicts in favor of newest remote version,
+* `Resolve to latest` if you want to resolve the conflicts in favor of latest document that was created
+
+## Related articles
+
+- [Server: Replication Bundle](../../../server/scaling-out/replication/how-replication-works)
+- [file-system studio: Conflicts View](../../../file-system/studio/synchronization/conflicts-view)
```
Comment (arekpalinski) [inline Documentation/3.5/Raven.Documentation.Pages/studio/overview/documents/conflicts-view.markdown]: file-system -> File System

File: Documentation/3.5/Raven.Documentation.Pages/studio/overview/documents/conflicts-view.markdown [hunk 3]
```diff
@@ -26,7 +26,7 @@ Then the conflict will disappear:
 
 ![Figure 5. Studio. No conflicts.](images/conflicts_view_5.png)
 
-Another way to solve all the issues in this page is to press the Reolve all conflicts button and chose how to resolve them:
+Another way to reolve all the issues in this page is to press the Reolve all conflicts button and chose how to resolve them:
```
Comment (arekpalinski) [inline Documentation/3.5/Raven.Documentation.Pages/studio/overview/documents/conflicts-view.markdown]: I meant: Reolve was a typo -> Resolve. Now we've got 2 typos in that sentence :)

Comment (arekpalinski) [inline Documentation/3.5/Raven.Documentation.Pages/studio/overview/documents/conflicts-view.markdown]: BTW I use the following spell checker in VS so it helps me to see typos immediately https://visualstudiogallery.msdn.microsoft.com/a23de100-31a1-405c-b4b7-d6be40c3dfff

**Comment by IdanHaim** [comment]
Didn`t notice that

**Pull request merged**